<?xml version="1.0"?>

<launch>
	<!-- Node for visualizing trajectory from turtlesim -->
 	<node name="robot_trajectory_visualization" pkg="tracking_visualization" type="trajectory_visualization.py" output="screen">
 		<remap from="odom_topic" to="/odometry/filtered"/>
 		<param name="frame_id" value="odom"/>
 		<param name="movement_threshold" value="0.05"/>
 		<param name="max_poses" value="500"/>
 	</node>

	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find tracking_visualization)/rviz/robot_odom_tracking.rviz" />

</launch>
